jans-cedarling/flask-sidecar/main/config.py

```
@@ -41,36 +44,49 @@ class BaseConfig:
+        "CEDARLING_BOOTSTRAP_CONFIG_FILE", "None"
+    )
     if CEDARLING_BOOTSTRAP_CONFIG_FILE == "None":
-        logger.info("Cedarling bootstrap file not found, falling back to environment variables")
+        logger.info(
+            "Cedarling bootstrap file not found, falling back to environment variables"
+        )
     else:
-        with open(CEDARLING_BOOTSTRAP_CONFIG_FILE, "r") as f:
-            CEDARLING_BOOTSTRAP_CONFIG = f.read()
+        try:
+            with open(CEDARLING_BOOTSTRAP_CONFIG_FILE, "r") as f:
```

Ruff (0.15.5)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)